fauxpose/README_fauxpose.md

```diff
@@ -0,0 +1,38 @@
+# FauxPose ILLIXR plugin
+
+## Summary
+The `FauxPose` ILLIXR plugin generates "fast_pose" data from a simple
+mathematical algorithm (circular movement).  The intent is for use when
+debugging other plugins and the developer wants a known pose trajectory
+without having to configure actual tracking.
+
+The movement is hard-coded to be on the Y=center.y plane, with the X and Z
+values set to rotate in a circle.  The period and amplitude of the movement
+are have default values (0.5, and 2.0 respectively) that can be changed
+through environment variables.
+
+    Topic details:
+    -   *Publishes* `pose_position` on `fast_pose` topic.
+
+## Usage
+The "FauxPose" plugin must be included in the YAML configuration file
+prior to any rendering plugin (such as "debugview" or "gldemo").  Also
```

Comment:
Will this work with monado (OXR) based applications?
If yes, could you change the config files to give that functionality as well?

Reply:
Seems like `configs/faux.yaml` is more of a sample to use, and documentation on how to use it should be sufficient.